src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$gap-size: 1px;
+$cells-in-row: 7;
+$height-cell: 100px;
+$width-cell: 100px;
+$week-days:
+  'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-cell * $cells-in-row + $gap-size * ($cells-in-row - 1);
+  gap: $gap-size;
+
+  &__day {
+    height: $height-cell;
+    width: $width-cell;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    border: 1px solid black;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $month-length from 28 through 31 {
+    &--month-length-#{$month-length} {
+      .calendar__day {
+        display: none;
+      }
+
+      @for $day from 1 through $month-length {
+        .calendar__day:nth-child(#{$day}) {
+          display: flex;
+        }
+
+        .calendar__day:nth-child(#{$day})::before {
```

Comment:
The `::before` pseudo-element is used here to add content, which is correct, but make sure that the content added is strictly for presentation purposes and not for adding essential information that should be in the HTML.